src/components/input/Input.js

```diff
@@ -168,6 +168,11 @@ Input.propTypes = {
    */
   value: PropTypes.string,
 
+  /**
+   * If true, the input is disabled and can't be clicked on.
```

Comment:
We're annoyingly inconsistent on using `true` and `True` in the descriptions. E.g. `in_navbar` on `DropdownMenu` has `Set this to True if the dropdown is inside a navbar. Default: False.`, but then all `disabled` props use the text you suggest. As a low-priority task, we could consistently use one or the other (I think I'm somewhat more in favour of True / False since the consumers of the documentation are Python users).

Reply:
Good point. I agree that using `True` / `False` makes the most sense.